rstudio/rstudio
# add select all command

### Intent

Addresses https://github.com/rstudio/rstudio/issues/11581.

### Approach

Adds a menu item for the Select All command (which seems to be the only documented way to implement custom application-specific keyboard shortcuts for Electron; at least as per https://www.electronjs.org/docs/latest/tutorial/keyboard-shortcuts)

### Automated Tests

Not included.

### QA Notes

Test that Cmd + A works in various contexts as described in https://github.com/rstudio/rstudio/issues/11581. Also check that Edit -> Select All functions as expected.

### Checklist
